Dockerfile

```diff
@@ -27,6 +27,4 @@ RUN bundle install --jobs 4 --retry 3
 VOLUME "${GEMSTASH_HOME}/data"
 
 EXPOSE 9292
```

Comment:
We should set the `USER` to `${GEMSTASH_USER}:${GEMSTASH_USER}`.

Reply:
Nice catch.
Done https://github.com/Intellection/docker-gemstash/pull/7/commits/587b2b75d627fd77bd222e066de2b6b23b0636e9.